infinispan-embedded/src/test/java/org/hibernate/ogm/datastore/infinispan/test/storedprocedures/InfinispanNamedParametersStoredProcedureCallTest.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.infinispan.test.storedprocedures;
+
+import java.util.List;
+import javax.persistence.EntityManager;
+import javax.persistence.ParameterMode;
+import javax.persistence.PersistenceException;
+import javax.persistence.StoredProcedureQuery;
+
+import org.hibernate.ogm.backendtck.storedprocedures.Car;
+import org.hibernate.ogm.utils.TestForIssue;
+import org.hibernate.ogm.utils.jpa.OgmJpaTestCase;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.fest.assertions.Assertions.assertThat;
+import static org.hibernate.ogm.backendtck.storedprocedures.Car.RESULT_SET_PROC_ID_PARAM;
+import static org.hibernate.ogm.backendtck.storedprocedures.Car.RESULT_SET_PROC_TITLE_PARAM;
+
+/**
+ * Testing call of stored procedures
+ *
+ * @author The Viet Nguyen &amp;ntviet18@gmail.com&amp;
+ */
+@TestForIssue(jiraKey = { "OGM-1429" })
+public class InfinispanNamedParametersStoredProcedureCallTest extends OgmJpaTestCase {
```

Comment:
I don't think these test are Infinipan specific, couldn't you update `NamedParametersStoredProcedureCallTest`.
This way we can also check othere dialects.

Reply:
We also run the common backendtck test defined in core for Infinispan. In this test we add new case when calling procedure by full qualified class name.